app/vep/vep_resources.py

```diff
@@ -140,3 +141,30 @@ async def download_results(request: Request, submission_id: str):
     except Exception as e:
         logging.debug(e)
         return response_error_handler(result={"status": 500})
+
+@router.get("/submissions/{submission_id}/results", name="view_results")
+async def fetch_results(request: Request, submission_id: str, page: int, page_size: int):
```

Comment:
In the search api, we named the parameter that is similar to the `page_size` parameter here, `per_page` ([link](https://gitlab.ebi.ac.uk/ensembl-web/ensembl_search_hub/-/blob/master/APISpecification.yaml?ref_type=heads#L85-105)). Is it worth using the same convention in this endpoint as well?

Reply:
Also, pages should start at 1, not at 0, as in the screenshot

![image](https://github.com/user-attachments/assets/222baf52-3842-450a-acb2-2a4bfd109a43)

Reply:
This is down to the vcf_results module. @jyothishnt do you want me to make a change in this pr to resolve it?

Reply:
yeah @jonk-ebi if you could, please commit to this branch.